bin/ex_doc

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env elixir
-Code.prepend_path Path.expand("../_build/dev/lib/earmark/ebin", __DIR__)
-Code.prepend_path Path.expand("../_build/dev/lib/ex_doc/ebin", __DIR__)
+Mix.start
```

Comment:
This is not really public API. If you want, you can rely on MIX_ENV but we should not start Mix.